tests/integration/database/migrations/issuer-field-migration.test.ts

```
@@ -0,0 +1,267 @@
+        SELECT issuer_id 
+        FROM assertions 
+        WHERE id = '33333333-3333-3333-3333-333333333333'
+      `);
+
+      expect(result.length).toBe(1);
+      expect(result[0].issuer_id).toBe('11111111-1111-1111-1111-111111111111');
+    });
+  });
+
+  describe('SQLite Migration', () => {
+    let db: ReturnType<typeof drizzleSqlite>;
```

ESLint

[error] 'db' is assigned a value but never used. Allowed unused vars must match /^_/u.

(@typescript-eslint/no-unused-vars)